src/app/examples/calendar-example/calendar-card-example.component.html

```diff
@@ -1,15 +1,18 @@
 <kirby-card>
     <kirby-calendar 
-        [disableWeekends]="true" 
-        [disablePastDates]="true" 
-        [disableDates]="disableDatesArray" 
-        [selectedDate]="selectedDate" 
+        [disableWeekends]="disableWeekends" 
+        [disablePastDates]="disablePastDates" 
+        [disableFutureDates]="disableFutureDates" 
+        [disabledDates]="setDisabledDates ? disabledDates : null"
+        [minDate]="setMinDate ? minDate : null"
+        [maxDate]="setMaxDate ? maxDate : null"
+        [selectedDate]="selectedDate"
         (dateChange)="onDateChange($event)">
     </kirby-calendar>
     
     <div>
         <button kirby-button (click)="selectNextMonth()">Next month</button>
-        <button kirby-button (click)="selectToday()" themeColor="secondary">Now</button>
+        <button kirby-button (click)="selectToday()" attentionLevel="2">Now</button>
```

Comment:
Hvor kommer "attentionLevel" lige fra, og hvad er "2" for et magisk tal?

Reply:
@ckibsen Det er den nye måde at style buttons i Kirby på. Jeg melder noget ud på Slack, når versionen bumpes i DRB